zerver/models/messages.py

```diff
@@ -186,6 +186,8 @@ class Meta:
                 # receiver.  The prefix of this index (realm_id,
                 # sender_id) can be used for scrubbing users and/or
                 # deleting users' messages.
+                # Also used to check welcome bot messages
+                # while onboarding
```

Comment:
This is not restricted to onboarding. Maybe simply: "Also used in 'send_welcome_bot_response'."